log-factory/sinks/log-to-slf4j/build.gradle

```diff
@@ -7,6 +7,5 @@ dependencies {
 	// internal logger, that is in service of the ServiceLoader - and if consumers should not need
 	// access to the internal logger.
 	implementation project(':log-factory')
-	compileOnly 'org.slf4j:slf4j-api:1.7.30' // users must provide their own impl and slf4j config
```

Comment:
We shouldn't be removing a compile dependency from here. The code still compiles b/c logback-classic transitively depends on slf4j-api, but we are best to be explicit here.

If you want to make a follow up ticket to consolidate slf4j versions to make sure they all match, go ahead and assign it to me.

Reply:
Done and done.